### DISCUSSION TITLE
关于“录播的mp4文件放在视频编辑软件中会产生往回跳帧情况”新发现

### DISCUSSION THREAD

**Discussion by Ponyonyope** [HMBSbige/BilibiliLiveRecordDownLoader#161] [Show and tell]
录制的flv如果经过本录播机转mp4则会出现问题
但是如果自己通过ffmpeg命令行，或者小丸工具箱等重新转封装成mp4则不会出现问题
使用的ffmpeg为同一个

**Comment by HMBSbige** [comment]
那可能是中间因为网络传输问题少了几帧（视频或者音频）导致的，可以关闭自动转封装，自行手动封装

话说可以在之前的 issue 下直接留言的不用新开一个

**Reply by Ponyonyope** [reply]
我又测试了一下，用其他录播软件录播的flv也一样，如果勾选修复时间戳就会出问题，不勾选就是正常的。
录制应该是没问题的，应该就是修复时间戳过程出现了问题。
![屏幕截图 2021-10-27 222715](https://user-images.githubusercontent.com/63969532/139086108-4c4774cd-664c-4b21-a3e0-7bdd0add1c7f.png)
第一行是不勾选修复时间戳的记录，第二三行是修复时间戳的记录。

**Reply by HMBSbige** [reply]
其实就是两种方法，毕竟收到的数据千奇百怪，用适合的就行了
